content/en/apps/guides/messaging/gateways/gateway/configuration.md

```diff
@@ -11,26 +11,26 @@ relatedContent: >
   apps/guides/messaging/gateways/gateway/phones
 ---
 
-Medic gateway supports Android 4.1 and above. To have it up and fully working, follow the below steps.
+CHT gateway supports Android 4.1 and above. To have it up and fully working, follow the 3 steps below.
 
-- Install the latest APK from the [releases](https://github.com/medic/cht-gateway/releases) in the `cht-gateway` repo.
+1. Install the latest APK from the [releases page](https://github.com/medic/cht-gateway/releases) in the `cht-gateway` repo.
 
-- Open the app.
-You will get a ```Warning:medic-gateway is not set as the default messaging app on this device``` if you are installing the app for the first time or afresh. Select ```HELP ME CHANGE``` and agree to the folloup system prompt about changing the default messaging app.
+2. Open the app.
+if you are installing the app for the first time or afresh, you will get a ```Warning:medic-gateway is not set as the default messaging app on this device``` . Select ```HELP ME CHANGE``` and agree to the follow-up system prompt about changing the default messaging app.
 
-- If you're configuring medic-gateway (v1.0.0 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```my_gateway_user``` and a password of ```topSecret```, fill in the settings as follows:
+3. Configure the app. If you're configuring cht-gateway(v1.00 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```gateway``` and a password of ```topSecret```, fill in the settings as follows:
```

Comment:
I think the version should be `1.0.0` like it was before. `1.00` looks weird to me!

```suggestion
3. Configure the app. If you're configuring cht-gateway(v1.0.0 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```gateway``` and a password of ```topSecret```, fill in the settings as follows:
```